middleware/ft800/lib/include/ft800_cmd.h

```diff
@@ -0,0 +1,738 @@
+ */
+#define FT800_CMD_DLSTART           0xFFFFFF00
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will start calibration
+ * routine.
+ */
+#define FT800_CMD_CALIBRATE         0xFFFFFF15
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a text.
+ */
+#define FT800_CMD_TEXT              0xFFFFFF0C
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a number.
+ */
+#define FT800_CMD_NUMBER            0xFFFFFF2E
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a button widget.
+ */
+#define FT800_CMD_BUTTON            0xFFFFFF0D
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a clock widget.
+ */
+#define FT800_CMD_CLOCK             0xFFFFFF14
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a gauge widget.
+ */
+#define FT800_CMD_GAUGE             0xFFFFFF13
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will fill area with gradient 
+ * color.
+ */
+#define FT800_CMD_GRADIENT          0xFFFFFF0B
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a keys.
+ */
+#define FT800_CMD_KEYS              0xFFFFFF0E
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a progress bar
+ * widget.
+ */
+#define FT800_CMD_PROGRESS          0xFFFFFF0F
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a slider.
+ */
+#define FT800_CMD_SLIDER            0xFFFFFF10
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a scroll bar
+ * widget.
+ */
+#define FT800_CMD_SCROLLBAR         0xFFFFFF11
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a dial widget.
+ */
+#define FT800_CMD_DIAL              0xFFFFFF2D
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will draw a toggle widget.
+ */
+#define FT800_CMD_TOGGLE            0xFFFFFF12
+
+/**
+ * @brief FT800 Co-Processor Objects Commands.
+ * @details When the co-processor engine executes this command, he will start track a press
+ * detection and movement at defined area.
+ */
+#define FT800_CMD_TRACK             0xFFFFFF2C
+
```

Comment:
Here doxygen groups should be ended like this:
```suggestion

  /*! @} */ // ft800_commands
 /*! @} */ // ft800
/*! @} */ // mwgroup

```